docs/sdk/src/guides/mod.rs

```diff
@@ -26,3 +26,6 @@ pub mod xcm_enabled_parachain;
 
 /// How to enable storage weight reclaiming in a parachain node and runtime.
 pub mod enable_pov_reclaim;
+
+/// Parachain guides
+pub mod parachains;
```

Comment:
For now I would suggest keeping the structure flat, as something like PoV reclaim is also about parachains. 

Although sadly the structure of the modules does break links, so I hope we can stick to one decision.